platform/Utc.roc

```diff
@@ -12,12 +12,13 @@ interface Utc
     imports [Effect, InternalTask, Task.{ Task }]
 
 ## Stores a timestamp as nanoseconds since UNIX EPOCH
-Utc := U128 implements [Inspect]
+Utc := I128 implements [Inspect]
 
 ## Duration since UNIX EPOCH
 now : Task Utc *
 now =
     Effect.posixTime
+    |> Effect.map Num.toI128
```

Comment:
I opted to do a typecast here, and leave Effect.posixTime typed as U128, since it appears that the rust SystemTime::now().duration_since()` is typed u128, so it makes sense to keep the Roc effect typed the same.